pandas-stubs/core/groupby/generic.pyi

```diff
@@ -300,9 +300,10 @@ class DataFrameGroupBy(GroupBy[DataFrame], Generic[ByT, _TT]):
         backend: str | None = ...,
         **kwargs,
     ) -> PlotAxes | Series: ...  # Series[PlotAxes]
+    # def size(self: DataFrameGroupBy[ByT, Literal[True]]) -> Series[int]: ...
```

Comment:
Delete the commented out code.

Reply:
My bad, I copied out the `size` method to make sure it was done right, should be good now!